src/ontime/core/plot/anomaly_plot.py

```diff
@@ -0,0 +1,212 @@
+import pandas as pd
+
+from ..time_series import TimeSeries, BinaryTimeSeries
+import altair as alt
+
+
+class AnomalyPlot:
+    _DATA_COLUMN_NAME = "data"
+
+    @staticmethod
+    def plot_anomalies(
```

Comment:
When I see the number of possible arguments in this function, it makes me think again that having an `AnomalyTimeSeries` object that is able to capture all kind of anomalies would make sense.